1. objects/1.md

```diff
@@ -1,73 +1,112 @@
 1. Pick a penguin from Wikipedia's List of Fictional Penguins (https://en.wikipedia.org/wiki/List_of_fictional_penguins) and create an object named `myPenguin` with properties that represent the information listed in each column on that Wikipedia page (for example: the character's name, origin, and author).
 
 ```js
+
 // your code goes here
+let myPenguin = {
+    character_name : "Whiteblack",
+    origin : "Whiteblack the Penguin Sees the World	",
+    author : "	H. A. Rey and Margret Rey",
+    notes : "Children's book about a travelling penguin"
+
+}
 ```
 
 2. Use console.log() to print the penguin's name to the console as part of a welcome message, like "Hello, I'm a penguin and my name is [NAME HERE]!"
 
 ```js
 // your code goes here
+console.log(`Hello, I'm a penguin and my name is ${myPenguin.character_name}`) ;
 ```
 
 3. Write another line of code that adds a new property to your penguin called canFly and set it to false.
 
 ```js
 // your code goes here
+myPenguin ['canFly'] = false;
 ```
 
 4. Add a method to your penguin called chirp that prints to the console: "CHIRP CHIRP! Is this what penguins sound like?"
 
 ```js
 // your code goes here
+myPenguin ['chirp'] = console.log ("CHIRP CHIRP! Is this what penguins sound like?")
```

Comment:
This will not add a method in the object. You need to wrap it inside a function.
```
myPenguin['chirp'] = function(){
  console.log ("CHIRP CHIRP! Is this what penguins sound like?")
}
```
What this will do is to store whatever gets returned from `console.log()` into the `chirp` key of the object. i.e `undefined`

Reply:
got it